src/libraries/System.Private.CoreLib/src/System/IO/StreamWriter.cs

```diff
@@ -804,60 +739,49 @@ public override Task WriteAsync(ReadOnlyMemory<char> buffer, CancellationToken c
                 return Task.FromCanceled(cancellationToken);
             }
 
-            Task task = WriteAsyncInternal(this, buffer, _charBuffer, _charPos, _charLen, CoreNewLine, _autoFlush, appendNewLine: false, cancellationToken: cancellationToken);
+            Task task = WriteAsyncInternal(buffer, appendNewLine: false, cancellationToken: cancellationToken);
             _asyncWriteTask = task;
             return task;
         }
 
         // We pass in private instance fields of this MarshalByRefObject-derived type as local params
         // to ensure performant access inside the state machine that corresponds this async method.
         // Fields that are written to must be assigned at the end of the method *and* before instance invocations.
```

Comment:
I guess this no longer applies?

Reply:
Right. Deleted. Thanks.

Reply:
Thank god the awful scourge of `MarshalByRefObject` is gone.